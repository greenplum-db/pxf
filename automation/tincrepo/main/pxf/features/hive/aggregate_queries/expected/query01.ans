-- start_ignore
-- end_ignore
-- @description query01 for HCatalog aggregate query
SELECT COUNT(*) FROM pxf_hive_small_data;
 count 
-------
    25
(1 row)