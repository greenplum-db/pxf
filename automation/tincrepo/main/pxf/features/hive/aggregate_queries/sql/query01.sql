-- @description query01 for Hive aggregate query
SELECT COUNT(*) FROM pxf_hive_small_data;