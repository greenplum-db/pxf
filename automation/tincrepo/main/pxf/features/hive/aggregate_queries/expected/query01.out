-- @description query01 for Hive aggregate query
SELECT COUNT(*) FROM pxf_hive_small_data;
 count 
-------
    25
(1 row)

